_includes/payment-order-other-features.md

```diff
@@ -1429,6 +1429,54 @@ with the request parameter `instrument`.
 
 {% include settlement-reconciliation.md api_resource="paymentorders" %}
 
+## 3-D Secure 2
+
+When dealing with card payments, 3-D Secure authentication of the
+cardholder is an essential topic. 3-D Secure 2 is an improved version of the
+old protocol, now allowing frictionless payments where transactions can be
+completed without input from the cardholder. Therefore, there are certain fields
+that should be included when implementing 3-D Secure 2. These are listed below
+in the abbreviated `Purchase` request. 
+
+{:.code-header}
+**Request**
+
+```http
+POST /psp/paymentorders HTTP/1.1
+Host: {{ page.api_host }}
+Authorization: Bearer <AccessToken>
+Content-Type: application/json
+
+{
+    "paymentorder": {
+        "operation": "Purchase",
+        "currency": "SEK",
+        "amount": 1500,
+        "vatAmount": 375,
+        "description": "Test Purchase",
+        "generatePaymentToken": true,
+        "userAgent": "Mozilla/5.0...",
+        "urls": {
+            "hostUrls": [ "https://example.com", "https://example.net" ]
+        },
+        "payer": {
+            "email": "olivia.nyhuus@payex.com",
+            "msisdn": "+4798765432",
+            "workPhoneNumber" : "+4787654321",
+            "homePhoneNumber" : "+4776543210"
+        }
+    }
+}
+```
+
+{:.table .table-striped}
+|     Required     | Field                             | Type         | Description                                                                                                                                                                                                                                                                                                                                                                                                   |
+| :--------------: | :-------------------------------- | :----------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+|                  | └─➔&nbsp;`email`                  | `string`     | The e-mail address of the payer. Will be used to prefill the Checkin as well as on the payer's profile, if not already set. Increases the chance for [frictionless 3-D Secure 2 flow](/{{ documentation_section }}/other-features#3-d-secure-2).                                                                                                                                         |
```

Comment:
The `paymentorder` and `payer` fields need to be added in the top of the table for the object hierarchy to make sense.